### DISCUSSION TITLE
How to make a swipeable animation like apple and whatsapp?

### DISCUSSION THREAD

**Discussion by hraschan** [software-mansion/react-native-gesture-handler#3042] [Q&A]
How would you implement the animation that Apple uses in its messenger or mail app or also whatsapp is using it in the chat list.
I tried to make it work using the swipleable but didn't manage.

Any tips on this one?

https://github.com/user-attachments/assets/165d1445-4ba9-4845-a962-e7ec7d5b627b

**Comment by latekvo** [comment]
Hey @hraschan,

we have implemented a very similar component [here](https://github.com/software-mansion/react-native-gesture-handler/blob/main/example/src/new_api/swipeable/AppleStyleSwipeableRow.tsx), let me know if it works for you.
